config.yml.example

```diff
@@ -6,7 +6,7 @@ user: git
 # You only have to change the default if you have configured Unicorn
```

Comment:
@david-martin I've removed the `config.yml`, as discussed in the previous iteration, but I've put the values from there in here. For OpenShift I'll try generating the config.yml in the template using a ConfigMap, and mounting that into place in the container.